e2e-tests/stdin-nested-config/test.sh

```
@@ -0,0 +1,27 @@
+set -euo pipefail
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)